src/generate_article_json.py

```diff
@@ -27,11 +27,13 @@ def render(path, json_output_dir):
         log = conf.multiprocess_log('generation.log', __name__)
         log.info(strbuffer.getvalue())
 
-def main(xml_dir, json_output_dir):
+def main(xml_dir, json_output_dir, num=None):
     paths = lmap(lambda fname: join(xml_dir, fname), os.listdir(xml_dir))
     paths = lfilter(lambda path: path.lower().endswith('.xml'), paths)
     paths = sorted(paths, reverse=True)
-    num_processes = 2
+    if num:
+        paths = paths[:num] # only scrape first n articles
+    num_processes = -1
```

Comment:
ah, this should be taken our from this branch and put in master. Being leftover there, it's probably why the bot-lax-adaptor build is taking so long

Reply:
> it's probably why the bot-lax-adaptor build is taking so long

yes, we made the change to throttle the rebuilding of the cache file so we didn't overwhelm iiif. I'll update master with this change directly if you haven't already